testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
+           originIp: origin.ip
+           source_user: origin.user
+           timestamp: request.time
+           method: request.method | ""
```

Comment:
this seems like an odd artifact to surface in the config file (and will quickly become out of date). is this just a workaround for #285 ? If so, can we add a comment about that so future me will remember?